### ISSUE TITLE
Document how to use negation in globs to ignore files in a CLI command

### ISSUE THREAD

**Issue by damianobarbati** [stylelint/stylelint#2676]
Can you please add the following argument to stylelint (like we have on eslint)?

Great work guys!

Syntax:
`--ignore-pattern [String]    Pattern of files to ignore (in addition to those in .stylelintignore)`

**Comment by hudochenkov** [comment]
You can ignore files from CLI like so:

```bash
stylelint "**/*.css, !**/docker/**"

stylelint "**/*.css" "!**/docker/**"
```

We should improve docs with these examples.

**Comment by ntwb** [comment]
Here's the docs that should document this:
• https://stylelint.io/user-guide/cli/ 
• Sourced for the above https://github.com/stylelint/stylelint/blob/master/docs/user-guide/cli.md

**Comment by damianobarbati** [comment]
Is this correct? 
`stylelint '**/*.css **/*.scss !**/*.min.css'  --ignore-path .gitignore`

Any chance to be symmetric to eslint adding the ignore-pattern? 🙇🏻
```
    "scripts": {
        "eslint": "./node_modules/eslint/bin/eslint.js **/*.js --ignore-path .gitignore --ignore-pattern **/*.min.js",
        "stylelint": "./node_modules/stylelint/bin/stylelint.js **/*.css **/*.scss --ignore-path .gitignore --ignore-pattern **/*.min.css"
     }
```

**Comment by hudochenkov** [comment]
@damianobarbati you forgot a comma:

```
stylelint '**/*.css, **/*.scss, !**/*.min.css' --ignore-path .gitignore
```

BTW, you can replace `./node_modules/eslint/bin/eslint.js` with just `eslint`, and `./node_modules/stylelint/bin/stylelint.js` with `stylelint`.

**Comment by damianobarbati** [comment]
Thanks @hudochenkov but I always keep everything scoped to project, no globals but yarn and pm2.

Same with commas? =>
```
imac:quant01 damz$ ./node_modules/stylelint/bin/stylelint.js '**/*.css, **/*.scss, !**/*.min.css' --ignore-path .gitignore
Error: **/*.css, **/*.scss, !**/*.min.css does not match any files
    at globby.then.filePaths (/Users/damz/Desktop/quant01/node_modules/stylelint/lib/standalone.js:124:33)
    at <anonymous>
```

**Comment by hudochenkov** [comment]
Could you create minimal reproducible test case repository, please?

```
"scripts": {
   "lint": "stylelint ."
}
```

`stylelint .` here is the same as `./node_modules/.bin/stylelint`. It runs locally installed stylelint. Same with other tools. https://docs.npmjs.com/misc/scripts#path

**Comment by damianobarbati** [comment]
oh great, I had already observed this behaviour but never found the relevant part on npm documentation so I tried to not rely on it: thanks so much @hudochenkov, a lot less chatting in my package.json then!
